unfetter-discover/unfetter-ui
# initial implementation of global feed. WIP.

Style carousel components to match mock-ups.

To Test:
- Make sure navigating to Threat Beta dashboard shows partial layout of global feed
- See that any created Threat Boards are listed on left side
- Verify that clicking on any of the links in the left side navigates appropriately to the board feed view.
- Verify that reports are visible in the recent reports carousel 

supports unfetter-discover/unfetter#1491
